### ISSUE TITLE
[Feature Request] Default Roll+ Mode

### ISSUE THREAD

**Issue by zero127** [p4535992/foundryvtt-better-rolltables#4]
A checkbox in the config to default when called to use the Roll+. 

I almost fully use BRT with Monks Active Tiles for "working treasure chests" in the world. (I have a ticket there as well asking for the Roll+ as a hook to choose as a tile trigger; i can see the value in separating them) 

When activated and Roll on a Table is used, it only does the Roll mode with no way to configure it. I'm not sure how the addon works, this may be impossible. If so, let me know and I will just close this. Thanks either way if you consider this.

**Comment by p4535992** [comment]
For that I would have to overwrite the foundryvtt code with a wrapper I'm thinking about whether or not to do that because it can conflict with other modules and foundryvtt. As a solution for now I opted for building an [API](https://github.com/p4535992/foundryvtt-better-rolltables/blob/master/wiki/api/api.md) (still writing...)

So for example for doing a roll using the brt code:

```
let tableHarvester = game.tables.get("NIHCtKDqCXpHTuNx");
game.modules.get("better-rolltables").api.betterTableRoll(tableHarvester, {
   rollMode: "gmroll",
   dc: 10,
   skill: "med"
});
```

if you have any special requests for the api let me know.

P.S. For the next release the 2.1.0 I rewrote and bug fixed a lot of stuff, so if you want to wait a few days to do the testing on your side...

**Comment by zero127** [comment]
makes sense. That's the stuff I wasn't sure - from a user point of view it seems easy to just "oh yeah do this by default" I imagine the solution may need to come from MATT then, to tap BRT's roll functions. Unless some plugin for both could provide that. Not sure. Worst case I can always just change it to a text message linking me as GM to the table to roll it. I just liked the old functionality where the players could open the chests themselves. But we'll adjust as needed. I'll see if i can work out what to do with this, though. The bug is in your ear at this point so if you go for it, awesome, if its too much work or not possible/smart to do so, then so be it; I certainly trust your judgement!  Thanks for the time either way.

**Comment by p4535992** [comment]
leave this open as a reminder for me if put  a wrapper or not in the future

**Comment by zero127** [comment]
Any luck on this one good sir? Or could i possibly trouble for a macro i could use to default my MATT tiles to run BRT rather than default roll?

**Comment by TheAtrivion** [comment]
I would very much like this feature too. Using TheRippers Hexplorer which automates tablerolls for each hex moved into, it sadly only rolls the normal Roll and ignores Roll+ which automatically can hide some rolled events to trigger. 
It would be a massive QOL to have a Setting which makes Roll+ the default roll for Foundry. =)
Thanks in advance /Atrivion
